### DISCUSSION TITLE
BlazorFluentUI Server side Textbox page is chatty

### DISCUSSION THREAD

**Discussion by gallwood** [BlazorFluentUI/BlazorFluentUI#370] [Q&A]
I have noticed the Textbox page on the Demo site for serverside Blazor is constantly posting back and forth to the server, even when there is no interaction with the page. Is this to be expected?

**Comment by limefrogyank** [comment]
Serverside demo is behind version-wise so I would not rely too much on it for up-to-date scenarios.

However, there are a few components that probably talk to the server a lot.  There's the data validation code and  the auto-adjusting textbox height that I can think of.  We'd definitely welcome help if you can narrow it down and submit a PR. :)
